sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/SessionTokenHelper.java

```diff
@@ -108,7 +112,16 @@ private static ISessionToken getLocalSessionToken(
                 } else {
                     highestSessionToken = highestSessionToken.merge(parsedSessionToken);
                 }
+            }
+        }
 
+        if (highestSessionToken == null) {
+            if (StringUtils.isNotEmpty(globalSessionToken)) {
+                logger.warn("The session token : {} for partition key range id : {} and collection rid : {} could not be evaluated, " +
```

Comment:
I am thinking usually this should only happen when it is the first time SDK sending requests to that partitionKeyRangeId? so not sure warning level is the one to be used here. 

And also in the diagnostics, we also log the request sessionToken SDK sent to the backend, will it be enough for debugging purpose?

Reply:
Discussed offline.